### PR TITLE
Feature/personal profile UI

### PR DESCRIPTION
**Pull request by WalterGoldenOwl**
Description Pull Request:
## Todo
- Apply UI applicant profile

## Task
Link of task:  https://trello.com/c/dNNgtgqd

## Screenshot
Screenshot_1697453647
Screenshot_1697453650
Screenshot_1697453665

## Video

https://github.com/dustin-goldenowl/jobspot/assets/144195318/e8fbf2ec-4621-488d-bb7b-2bcbed382bee

### CODE REVIEW

File: lib/src/presentations/applicant_profile/screen/applicant_profile_view.dart [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+            ),
+            Padding(
+              padding: const EdgeInsets.all(AppDimens.smallPadding),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  const SizedBox(height: 40),
+                  ClipOval(
+                    child: CachedNetworkImage(
+                      imageUrl: PrefsUtils.getUserInfo()!.avatar,
```
Comment (dustin-goldenowl) [inline lib/src/presentations/applicant_profile/screen/applicant_profile_view.dart]: You should separate and name a variable for this line. And must check null for this case

File: lib/src/presentations/applicant_profile/screen/applicant_profile_view.dart [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+                          SvgPicture.asset(AppImages.logo),
+                    ),
+                  ),
+                  const SizedBox(height: 7),
+                  Text(
+                    PrefsUtils.getUserInfo()?.name ?? "",
+                    style: AppStyles.boldTextWhite.copyWith(fontSize: 16),
+                  ),
+                  Text(
+                    PrefsUtils.getUserInfo()!.address.isEmpty
```
Comment (dustin-goldenowl) [inline lib/src/presentations/applicant_profile/screen/applicant_profile_view.dart]: The same above

File: lib/src/presentations/applicant_profile/screen/applicant_profile_view.dart (hunk 1 above)
Comment (WalterGoldenOwl) [reply lib/src/presentations/applicant_profile/screen/applicant_profile_view.dart]: I fixed it

File: lib/src/presentations/applicant_profile/screen/applicant_profile_view.dart (hunk 2 above)
Comment (WalterGoldenOwl) [reply lib/src/presentations/applicant_profile/screen/applicant_profile_view.dart]: I fixed it

**Pull request merged**